justfile

```diff
@@ -79,7 +79,7 @@ runtime target=default_deploy_target platform=default_platform:
         --add-namespace-object \
         --node-installer-target-conf-type ${node_installer_target_conf_type} \
         --platform {{ platform }} \
-        runtime > ./{{ workspace_dir }}/runtime/runtime.yml
+        runtime | yq -P > ./{{ workspace_dir }}/runtime/runtime.yml
```

Comment:
Is this still required with the changes to `kuberesource/writer.go`? I'd rather fix this in Go, if possible.